README.md

```diff
@@ -100,14 +100,40 @@ or
   git push -u origin main
 ```
 
-If you get an error message like the one below, you probably forgot to fork the repository before cloning it. It is best to start over and fork the project repository first.
+If you get an error message like the one below, you probably forgot to fork the repository before cloning it.
 
 ```bash
 ERROR: Permission to EddieHubCommunity/hacktoberfest-practice.git denied to <your-github-username>.
 fatal: Could not read from remote repository.
 Please make sure you have the correct access rights and the repository exists.
 ```
 
+No need to panic or loose your streak at this point. Follow along to fix this issue.
+
+To fix this issue, fork the project now using the gray `Fork` button in the top right of this page.
```

Comment:
We could provide a link to fork the repository directly.
```suggestion
To fix this issue, fork the project using the gray [`Fork`](https://github.com/EddieHubCommunity/hacktoberfest-practice/fork) button in the top right of this page.
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant